Branches/Dev/core/obj_Agents.iss

```diff
@@ -720,6 +720,51 @@ objectdef obj_Agents
 		Missions.MissionCache:SetLowSec[${amIterator.Value.AgentID},${MissionParser.IsLowSec}]
 	}
 
+	function:bool CheckButtonExists(string buttontext)
```

Comment:
Functions are for this that have to wait across frames; this should be a member

Reply:
Same with the other new function tems here, but the pressbuttoms should be methods